cmd/registry/cmd/resolve/resolve.go

```diff
@@ -64,19 +64,15 @@ func Command(ctx context.Context) *cobra.Command {
 
 			log.Printf("Generated %d actions. Starting Execution...", len(actions))
 
-			taskQueue := make(chan core.Task, 1024)
-			for i := 0; i < 64; i++ {
-				core.WaitGroup().Add(1)
-				go core.Worker(ctx, taskQueue)
-			}
-			defer core.WaitGroup().Wait()
-			defer close(taskQueue)
-
 			// Submit tasks to taskQueue
-			for i, a := range actions {
-				taskQueue <- &controller.ExecCommandTask{
-					Action: a,
-					TaskID: fmt.Sprintf("task%d", i),
+			for _, a := range actions {
+				log.Printf("Executing action: %s", a)
+				rootCmd := cmd.Root()
+				rootCmd.SetArgs(strings.Split(a, " "))
+				log.Printf("%s", rootCmd.CalledAs())
```

Comment:
Does this ever print anything? Since the command hasn't been called yet, the [documentation](https://pkg.go.dev/github.com/spf13/cobra#Command.CalledAs) says it should be an empty string. Please remove this or add some explanation to the message if it's going to log empty lines.

Reply:
I added it for debugging my test. Missed removing it, there is no need to log `CalledAs()`. Thanks for pointing it out!